flytekit/image_spec/image_spec.py

```diff
@@ -181,6 +184,15 @@ def with_apt_packages(self, apt_packages: Union[str, List[str]]) -> "ImageSpec":
 
         return new_image_spec
 
+    def force_push(self, is_force_push: Optional[bool] = True) -> "ImageSpec":
+        """
+        Builder that returns a new image spec with force push enabled.
+        """
+        new_image_spec = copy.deepcopy(self)
+        new_image_spec.is_force_push = is_force_push
```

Comment:
```suggestion
        self._is_force_push = True
```

Reply:
```
new_image_spec = copy.deepcopy(self)
new_image_spec.is_force_push = is_force_push
```

We need to carry these parameters because it is possible to use other methods and combinations together